winlink_monitor.py

```diff
@@ -328,38 +353,49 @@ def history_string(history):
     return ret
 
 
-
 def diff_and_report_health_state(old, new):
     for (node, health) in new.items():
-        if (old[node] != health):
-            # TODO - Real reporting
```

Comment:
That TODO was my plan for tomorrow

(To be fair, I was just gonna dupe the logging line with a syslog() call.)

Reply:
I have been making the assumption that the calls to logging are being sent to syslog.  Lets just make sure that is true in setup (or earlier) with the potential to send to STDERR as well.

Reply:
No, logging doesn't do syslog by default.

I was going to be more judicious with the syslog stuff than the rest of the logging.